EsApi4DScheduleSampleApp/EsApi4DScheduleSampleApp/Program.cs

```diff
@@ -0,0 +1,119 @@
+﻿/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+using System;
+using System.Diagnostics.Metrics;
+using System.Net;
+using System.Net.Http.Json;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using EsApi4DScheduleSampleApp;
+
+
+await ConsoleApp.RunAsync(args, async (arguments, configuration) =>
+{
+    var client = new HttpClient()
+    {
+        BaseAddress = configuration.ServiceHost,
+        DefaultRequestHeaders = { { "Authorization", $"Bearer {arguments.Token}" } },
+    };
+
+    ConsoleApp.Log("Initial query to test authorization. Fetching schedules for a given project - Queries all schedules in the specified Project.");
+    ConsoleApp.Log($"Sending GET request to {client.BaseAddress}?projectId={arguments.Schedule}");
+    var scheduleResponse = await client.GetAsync($"?projectId={arguments.Schedule}");
+    var stringResp = scheduleResponse.Content.ReadAsStringAsync().Result;
+    Console.WriteLine($"Response: {stringResp}");
+    Console.WriteLine();
+
+    // only check once for authentication problems
+    if (!scheduleResponse.IsSuccessStatusCode)
+        {
+            ConsoleApp.Log("Fetching schedules for given project failed with status code: {0}.", scheduleResponse.StatusCode);
+            if (scheduleResponse.StatusCode is HttpStatusCode.Unauthorized or HttpStatusCode.Forbidden && scheduleResponse.Headers.TryGetValues("WWW-Authenticate", out var reasonHeader))
+            {
+                ConsoleApp.Log("Authentication failure reason: {0}", string.Join(";", reasonHeader));
+            }
+            return;
+        }
+
+    ConsoleApp.Log("Authentication successful");
+    Console.WriteLine();
+
+    if (arguments.Single)
+    {
+        if (!stringResp.Contains("\"items\":[]"))
+        {
+            ConsoleApp.Log("Fetching single schedule - Use this endpoint to query a single schedule in the specified project schedule list.");
+            ConsoleApp.Log($"Sending GET request to {client.BaseAddress}{arguments.Schedule}");
+            var get = new HttpGet($"{arguments.Schedule}", client);
+            var response = get.Get().Result;
```

Comment:
Why are you using .Result in async context? Just await get.Get();

Reply:
done in most recent commit